homeassistant/components/matter/sensor.py

```diff
@@ -462,4 +468,19 @@ def _update_from_device(self) -> None:
         required_attributes=(clusters.Switch.Attributes.CurrentPosition,),
         allow_multi=True,  # also used for event entity
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SENSOR,
+        entity_description=MatterSensorEntityDescription(
+            key="MicrowaveOvenMode",
+            translation_key="microwave_oven_mode",
+            device_class=SensorDeviceClass.ENUM,
+            state_class=None,
+            # convert to set first to remove the duplicate unknown value
```

Comment:
This comment doesn't match reality. There are no duplicates at the moment. There could be in the future if we extend the map.